src/coredata.cs

```diff
@@ -2413,4 +2490,92 @@ interface NSConstraintConflict
 #endif
 		NSDictionary[] ConflictingSnapshots { get; }
 	}
+
+	[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+	[BaseType (typeof(NSPersistentStoreRequest))]
+	[DisableDefaultCtor] // designated
+	interface NSBatchInsertRequest {
+
+		[DesignatedInitializer]
+		[Export ("init")]
+		IntPtr Constructor ();
+
+		[Export ("entityName")]
+		string EntityName { get; }
+
+		[NullAllowed, Export ("entity", ArgumentSemantic.Strong)]
+		NSEntityDescription Entity { get; }
+
+		[NullAllowed, Export ("objectsToInsert", ArgumentSemantic.Copy)]
+		NSDictionary<NSString, NSObject>[] ObjectsToInsert { get; set; }
+
+		[Export ("resultType", ArgumentSemantic.Assign)]
+		NSBatchInsertRequestResultType ResultType { get; set; }
+
+		[Static]
+		[Export ("batchInsertRequestWithEntityName:objects:")]
+		NSBatchInsertRequest BatchInsertRequest (string entityName, NSDictionary<NSString, NSObject>[] dictionaries);
+
+		[Export ("initWithEntityName:objects:")]
+		IntPtr Constructor (string entityName, NSDictionary<NSString, NSObject>[] dictionaries);
+
+		[Export ("initWithEntity:objects:")]
+		IntPtr Constructor (NSEntityDescription entity, NSDictionary<NSString, NSObject>[] dictionaries);
+	}
+
+	[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+	[BaseType (typeof(NSPersistentStoreResult))]
+	interface NSBatchInsertResult {
+		[NullAllowed, Export ("result", ArgumentSemantic.Strong)]
+		NSObject Result { get; }
+
+		[Export ("resultType")]
+		NSBatchInsertRequestResultType ResultType { get; }
+	}
+
+	[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+	[BaseType (typeof(NSAttributeDescription))]
+	interface NSDerivedAttributeDescription : NSSecureCoding {
+		[NullAllowed, Export ("derivationExpression", ArgumentSemantic.Strong)]
+		NSExpression DerivationExpression { get; set; }
+	}
+
+	[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+	[BaseType (typeof(NSPersistentContainer))]
+	[DisableDefaultCtor]
+	interface NSPersistentCloudKitContainer {
+
+		[Export ("initWithName:managedObjectModel:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string name, NSManagedObjectModel model);
+
+		[Export ("initializeCloudKitSchemaWithOptions:error:")]
+		bool Create (NSPersistentCloudKitContainerSchemaInitializationOptions options, [NullAllowed] out NSError error);
```

Comment:
`Create` what ? not sure it's the right name since it does not return any new instance
maybe `Initialize` ?